requirements.txt

```diff
@@ -0,0 +1,9 @@
+psutil
+ninja
+pyyaml
+fasm
+git+https://github.com/SymbiFlow/prjxray.git
```

Comment:
FYI - We should really figure out how to make this not needed here. The Python stuff from prjxray should probably be coming in via either the `prjxray-tools` or the `xc-fasm` repositories.....

Reply:
I guess one possibility would be to publish on PyPI the prjxray library, so that it can safely be added as an install_dependency when getting xc-fasm.

Otherwise, I guess it could be doable to get the python library within the prjxray-tools, which was initially thought to carry only the compiled prjxray binaries.